src/Forwarder.h

```diff
@@ -83,7 +83,7 @@ class Forwarder {
   ConversionScheduler conversion_scheduler;
   std::atomic<ForwardingStatus> forwarding_status{ForwardingStatus::NORMAL};
   std::shared_ptr<KafkaW::Producer> status_producer;
-  std::unique_ptr<KafkaW::ProducerTopic> status_producer_topic;
+  std::shared_ptr<KafkaW::ProducerTopic> status_producer_topic;
```

Comment:
Does `Forwarder` need `status_producer_topic` or could it move the unique pointer (pass ownership) to the status reporter?